apps/libs/actuator/src/index.ts

```diff
@@ -1,3 +1,6 @@
 export * from './ActuatorModule'
 export * from './ActuatorController'
 export * from './BlockchainCppModule'
+export * from './filters/ErrorFilter'
+export * from './filters/GlobalValidationPipe'
+export * from './filters/ResponseInterceptor'
```

Comment:
And this.

Also instead of putting this into the `actuator` module, you should create a new module called `libs/filters`?